libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -697,7 +697,7 @@ AP_AHRS_DCM::drift_correction(float deltat)
         }
         float airspeed;
         if (airspeed_sensor_enabled()) {
-            airspeed = _airspeed->get_airspeed();
+            airspeed = AP::airspeed()->get_airspeed();
```

Comment:
Needs a null check. Not all vehicles instantiate airspeed

Reply:
airspeed_sensor_enabled() does that.  Further, we're already dereferencing a pointer :-)

I do want to rename `airspeed_sensor_enabled()` to `using_airspeed_sensor()`.

Reply:
I meant AP_::airspeed() could be nullptr

Reply:
No, it can't; `airspeed_sensor_enabled` checks for nullptr.  It already checks `_airspeed` for nullptr, and has been changed to check the singleton instead.

Reply:
This is the same argument that was discussed on yesterday's dev call. The function should return the result when true, in this case a non-null ptr. Having a function check something, and then at a later time it is assumed true. That's not thread safe.